### DISCUSSION TITLE
Terrain3D exaggerate vector tiles

### DISCUSSION THREAD

**Discussion by wiesehahn** [maplibre/maplibre-gl-js#1245] [General]
The Terrain3D example uses raster tiles to exaggerate 
(https://maplibre.org/news/2022-05-20-terrain3d/#12/47.27574/11.39085/0/52)

Is this or will this also be possible on vector tiles?

**Comment by wiesehahn** [comment]
It seems to work with vector tiles as well 
(https://www.maptiler.com/news/2022/05/maplibre-2/)

**Comment by HarelM** [comment]
Vector tiles are supported with terrain 3D. Having said that the vector tiles are converted to an image and this image is stretched over the terrain 3D surface which creates some awkward looking 3D maps, since it's no longer as sharp as it used to be for vector tiles.
I suggest to simply try it out and see if it fits your needs.
